Week 7/RecipeApp/app/src/main/java/com/janewaitara/movieapp/model/Recipe.kt

```diff
@@ -0,0 +1,56 @@
+package com.janewaitara.movieapp.model
+
+import android.os.Parcel
+import android.os.Parcelable
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import kotlinx.serialization.Serializable
+
+@Serializable
+@Entity(tableName = "recipe_table")
+data class Recipe(
+    @PrimaryKey val id: Int,
```

Comment:
As Filip discussed, different model classes can be provided for different parts of the project to reduce the number of tags, but overall good work on this model class.